lib/web/desktop_playback.go

```diff
@@ -182,7 +183,13 @@ func (pp *playbackPlayer) streamSessionEvents(ctx context.Context, cancel contex
 					time.Sleep(time.Duration(e.DelayMilliseconds-lastDelay) * time.Millisecond)
 					lastDelay = e.DelayMilliseconds
 				}
-				if _, err := pp.ws.Write(e.Message); err != nil {
+				// Note that e.Message is a []byte, which will be marshaled as a base64 encoded string
```

Comment:
nit: I'd be OK ditching this comment, it is a default behavior after all. Your call.

Reply:
+1